### PR TITLE
Fix Skript MinecraftServer.isRunning reflection on 1.20(.1)

### PR DESCRIPTION
**Pull request by C0D3-M4513R**
Related #6351 

Adds the obfuscated name for Server#isRunning for 1.20 versions.

Please note that whilst this change is untested, it should be correct.

### CODE REVIEW

**Comment by sovdeeth** [comment]
Does it fix your issue when you test?

**Comment by C0D3-M4513R** [comment]
I have made other changes to Ketting (adding the bukkit method back in). So even if I did compile this (which I didn't) it wouldn't test the relavent code paths.

**Review by sovdeeth — APPROVED** [review]
Can replicate with Spigot servers, this does fix it.

**Comment by skmedix** [comment]
What if the mappings was saved in a json file (or any other format) in the jar, so you don't have to change the code every major version?

**Comment by sovdeeth** [comment]
> What if the mappings was saved in a json file (or any other format) in the jar, so you don't have to change the code every major version?

Then we'd just have to change the mappings every version, no? It seems like the same outcome to me.
A better solution would be figuring out how to remove this dependency on NMS entirely.

**Comment by skmedix** [comment]
> Then we'd just have to change the mappings every version, no? It seems like the same outcome to me. A better solution would be figuring out how to remove this dependency on NMS entirely.

Yeah, fair enough. My idea was to just have a list of mappings, instead of adding another round of if statements. Kinda cleaner, but you still need to bump the list when needed, but I guess it's fine for now

**Comment by sovdeeth** [comment]
> > Then we'd just have to change the mappings every version, no? It seems like the same outcome to me. A better solution would be figuring out how to remove this dependency on NMS entirely.
> 
> Yeah, fair enough. My idea was to just have a list of mappings, instead of adding another round of if statements. Kinda cleaner, but you still need to bump the list when needed, but I guess it's fine for now

Ah I see what you mean. Sure, I think if we need to touch this again we should swap to a switch or a map like your suggestion, but I think we should aim to get rid of it by then if possible.

File: src/main/java/ch/njol/skript/Skript.java [hunk 1]
```diff
@@ -1157,7 +1157,11 @@ public void onPluginDisable(PluginDisableEvent event) {
 			try {
 				// Spigot removed the mapping for this method in 1.18, so its back to obfuscated method
 				// 1.19 mapping is u and 1.18 is v
-				String isRunningMethod = Skript.isRunningMinecraft(1, 19) ? "u" : Skript.isRunningMinecraft(1, 18) ? "v" :"isRunning";
+				String isRunningMethod = "isRunning";
+
+				if (Skript.isRunningMinecraft(1, 20)) isRunningMethod = "v";
+				else if (Skript.isRunningMinecraft(1, 19)) isRunningMethod = "u";
+				else if (Skript.isRunningMinecraft(1, 18)) isRunningMethod = "v";
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/Skript.java:1164]:
```suggestion
				if (Skript.isRunningMinecraft(1, 20)) {
					isRunningMethod = "v";
				} else if (Skript.isRunningMinecraft(1, 19)) {
					isRunningMethod = "u";
				} else if (Skript.isRunningMinecraft(1, 18)) {
					isRunningMethod = "v";
				}
```

**Review by APickledWalrus — APPROVED** [review]

**Pull request merged**